src/sgame/sg_admin.cpp

```diff
@@ -2420,9 +2420,8 @@ bool G_admin_slap( gentity_t *ent )
 	// using knockback in ent->Damage() won't throw the victim
 	// about if the damage caused to them is 0.
 	vec3_t kvel;
-	float mass = BG_Class( vic->client->pers.classSelection )->health;
```

Comment:
To be clear, slap velocity is dependent on *health* not mass, which is another thing. Imo, I would rather just have slap take another param for velocity and by default use the existing velocity.

Reply:
Yeah, I think health was just an approximation for mass in that case. Problem with second parameter is that the second (optional) parameter already reduces health, that is important to kill stuck bots. Let me see what I can do...

Reply:
Add a 3rd param? `/slap badbot 0 100000`

Reply:
I agree.